nav2_toolkit/launch/pose_saver.launch.py

```diff
@@ -0,0 +1,25 @@
+from launch import LaunchDescription
+from launch.actions import DeclareLaunchArgument
+from launch.substitutions import LaunchConfiguration
+from launch_ros.actions import Node
+
+def generate_launch_description():
+    return LaunchDescription([
+        DeclareLaunchArgument('auto_start_saving', default_value='true'),
```

Comment:
If declared here, it doesn't need to be declared in the `nav2_bringup` file as well.